src/hooks/makeMutation.ts

```diff
@@ -66,7 +66,7 @@ export function useMutation<TData, TVariables>(
     },
     onError: (err: ApolloError) => {
       if (!disableErrorHandling) {
-        if (err.graphQLErrors) {
+        if (err?.graphQLErrors.length > 0) {
```

Comment:
Since `err.graphQLErrors` returns an empty array this if was always truthy

Reply:
are you sure about that?

Reply:
Well, it will be good that @wojteknowacki somehow test this :)